enterprise/cmd/executor/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-# This script builds the executor google cloud image.
+# This script builds the executor image as a GCP boot disk image and as an AWS AMI.
 
 cd "$(dirname "${BASH_SOURCE[0]}")"
 set -eu
```

Comment:
```suggestion
set -Eeuo pipefail
```

Also, why is this not at the top?

Reply:
This preamble is the same on all build.sh scripts. Should we change them everywhere?

Reply:
Probably yes, from what I recall `set -Eeuo pipefail` is the set of "strict" flags that you usually want to apply. But, it's possible that this would break some scripts so doing this everywhere might not be trivial.

Reply:
But then this shouldn't block this PR, right?

Reply:
Correct.